docs/data/charts/export/inflationData.js

```diff
@@ -0,0 +1,75 @@
+/* Source: https://data.worldbank.org/indicator/FP.CPI.TOTL.ZG?end=2024&locations=DE&start=1960&view=chart */
```

Comment:
Nice data. What about moving them in `docs/data/charts/dataset` to make it easier to reuse it from other pages?